modules/custom/az_dam/az_dam.info.yml

```
@@ -0,0 +1,6 @@
+name: 'Quickstart DAM'
+type: module
+core_version_requirement: ^9
+package: The University of Arizona
+dependencies:
+  - intelligencebank:intelligencebank
```

I think this needs to be `ib_dam`.  `intelligencebank` is the package name and it contains submodules.
```suggestion
  - intelligencebank:ib_dam
```

---

We may also want to add these as dependencies to `az_dam`:

```
  - intelligencebank:ib_dam_media
  - intelligencebank:ib_dam_wysiwyg
```